app-office/calcurse/calcurse-4.7.1.ebuild

```
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{8..10} )
```

```suggestion

PYTHON_COMPAT=( python3_{8..10} )
```

Nit: new line after EAPI

---

Resolved